tdrs-frontend/Dockerfile.dev

```diff
@@ -7,7 +7,7 @@ WORKDIR /home/node/app
 COPY --chown=node:node package.json yarn.lock ./
```

Comment:
This change already went into HHS main earlier today, but the Git History doesn't seem to recognize the change as the same. Again, this won't change anything in the current codebase.

Reply:
👍🏾